server.js

```diff
@@ -10,6 +10,7 @@ var routeHandler = require('./src/routeHandler');
 var PORT = 8000;
 var VAULT_URL_DEFAULT = process.env.VAULT_URL_DEFAULT || "";
 var VAULT_AUTH_DEFAULT = process.env.VAULT_AUTH_DEFAULT || "GITHUB";
+var VAULT_SUPPLIED_TOKEN_HEADER = process.env.VAULT_SUPPLIED_TOKEN_HEADER || undefined;
```

Comment:
No need for the `|| undefined` here since it will be set to `undefined` anyways if it DNE.